Description: support vdr channel detection for HEVC and HEVC-UHD channels
Origin: https://www.vdr-portal.de/forum/index.php?thread/133266-patch-hevc-support-in-sdt-c/
Author: HelmutB (vdr-portal.de)

Index: vdr-V20401-19-g196785ff/sdt.c
===================================================================
--- vdr-V20401-19-g196785ff.orig/sdt.c	2020-06-10 17:47:53.423693420 +0200
+++ vdr-V20401-19-g196785ff/sdt.c	2020-06-10 17:47:53.419693506 +0200
@@ -113,6 +113,8 @@
                    case 0x05: // NVOD time-shifted service
                    case 0x16: // digital SD television service
                    case 0x19: // digital HD television service
+                   case 0x1F: // HEVC digital television service
+                   case 0x20: // HEVC UHD digital television service
                         {
                         char NameBuf[Utf8BufSize(1024)];
                         char ShortNameBuf[Utf8BufSize(1024)];
